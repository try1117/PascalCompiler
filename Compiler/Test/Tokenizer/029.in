a.5(*commment
with
asterisk*)
integer double
