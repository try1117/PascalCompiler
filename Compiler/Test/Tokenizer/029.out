1  | 1  | VARIABLE                 | a                        | 
1  | 2  | SEP_DOT                  | .                        | 
1  | 3  | CONST_INTEGER            | 5                        | 5
4  | 1  | KEYWORD_INTEGER          | integer                  | 
4  | 9  | KEYWORD_DOUBLE           | double                   | 
